### ISSUE TITLE
Postico

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-postgresql#29]
@roryc89 / @Conorc1000,
you gents were using Postico for Postgres GUI, right?
https://eggerapps.at/postico

**Comment by roryc89** [comment]
@nelsonic yes we mostly use Postico and occasionally pgAdmin for a couple of tasks that Postico either doesn't do or we haven't discovered how to with Postico. 

I would definitely recommend Postico over pgAdmin as it has a more user friendly interface and is less buggy.

**Comment by jackcarlisle** [comment]
+1 for postico

**Comment by nelsonic** [comment]
@roryc89 + @jackcarlisle thanks for confirming. 👍 
Now I just need one that our Linux/BSD friends (_who don't have Macs_) can use ... 😉 
(_but for my work **today** I'm using Postico... and it's **way better** than pgAdmin! thanks again!_)

**Comment by nelsonic** [comment]
@roryc89 saw you using https://www.pgadmin.org v4 last Friday ...
are you still using Postico? or have you switched?
